service-deployment/bootstrap/prometheus/cleanup.sh.template

```diff
@@ -26,3 +26,12 @@ configmap/prometheus-configmap"
 for instance in ${INSTANCES}; do
   kubectl delete --ignore-not-found --now ${instance}
 done
+
+{% for host in machinelist %}
+    {% if 'prometheus' in machinelist[ host ] and machinelist[ host ][ 'prometheus' ] == 'true' %}
```

Comment:
I found your label of prometheus and node-exporter share the same key? So prometheus and node-exporter can't deployed to the same node.

Reply:
refined